packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -352,6 +366,16 @@ const CreatePassword = ({ password, platform, validatePassword, onSubmit, error_
                                 )}
                             </PasswordMeter>
                         </div>
+                        {is_real_financial_stp && !has_mt5_account && !is_bvi && (
```

Comment:
since we are going to show this regardless client has other mt5 account or not, don't need to check `has_mt5_account`
```suggestion
                        {is_real_financial_stp &&  !is_bvi && (
```